app/src/main/java/com/simplecity/amp_library/playback/MultiPlayer.java

```diff
@@ -53,7 +55,12 @@ private boolean setDataSourceImpl(final MediaPlayer mediaPlayer, final String pa
             } else {
                 mediaPlayer.setDataSource(path);
             }
-            mediaPlayer.setAudioStreamType(AudioManager.STREAM_MUSIC);
+            if (ShuttleUtils.hasOreo()) {
```

Comment:
Good change, but why not make this under a separate commit? Totally unrelated to the other changes.